util/config/schema.py

```diff
@@ -723,6 +723,12 @@
             "description": "The endpoint for the V4 security scanner",
             "x-example": "http://192.168.99.101:6060",
         },
+        "SECURITY_SCANNER_V4_CACHE_URL": {
+            "type": ["string", "null"],
+            "pattern": "^(rediss?|unix)://.+$",
```

Comment:
Just trying to understand the regex for the pattern. Can there be 2 consecutive "s" here? Or can it be just `(redis|unix)`?

Reply:
yeah, there's `redis` (plaintext) and `rediss` (tls). It's weird but in the [IANA registry](https://www.iana.org/assignments/uri-schemes/prov/rediss).